lectures/money_inflation_nonlinear.md

```diff
@@ -11,11 +11,19 @@ kernelspec:
   name: python3
 ---
 
+<<<<<<< HEAD
```

Comment:
@SylviaZhaooo this conflict hasn't been resolved. There are still git markups. Thanks.